DefaultConfig.json

```diff
@@ -1,5 +1,5 @@
 {
     "generatorOptions": {
-        "target": "psr2"
+        "target": "MODE_PSR2"
```

Comment:
Has to be changd in the `vog_config.json` as well

Reply:
done as well.